src/viz/Ensight_Translator.i.hh

```diff
@@ -49,19 +50,23 @@ template <typename SSF>
 Ensight_Translator::Ensight_Translator(
     const std_string &prefix, const std_string &gd_wpath,
     const SSF &vdata_names, const SSF &cdata_names, const bool overwrite,
-    const bool static_geom, const bool binary, const double reset_time)
+    const bool static_geom, const bool binary, const bool decomposed,
+    const double reset_time)
     : d_static_geom(static_geom), d_binary(binary), d_dump_dir(gd_wpath),
       d_num_cell_types(0), d_cell_names(), d_vrtx_cnt(0), d_cell_type_index(),
       d_dump_times(), d_prefix(), d_vdata_names(vdata_names),
       d_cdata_names(cdata_names), d_case_filename(), d_geo_dir(),
       d_vdata_dirs(), d_cdata_dirs(), d_geom_out(), d_cell_out(),
-      d_vertex_out() {
+      d_vertex_out(), d_decomposed(decomposed) {
+  // Insist(!d_static_geom,
+  //        "We no longer support static geometry (only one geometry file "
+  //        "created per time step) just set this to false");
```

Comment:
What is this guy about? :)

Reply:
Ah... I was going to prevent this option, but decided to leave it in. I'll remove it.